tensorboard/plugins/histogram/tf_histogram_dashboard/tf-histogram-dashboard.html

```diff
@@ -41,21 +41,29 @@
   <template>
     <tf-dashboard-layout>
       <div class="sidebar">
+        <div class="sidebar-section">
+          <div class="line-item">
+            <paper-checkbox
+                id="show-download-links"
+                checked="{{_showDownloadLinks}}"
+            >Show data download links</paper-checkbox>
+          </div>
+        </div>
         <div class="sidebar-section">
           <tf-option-selector
-            id="histogramModeSelector"
-            name="Histogram mode"
-            selected-id="{{_histogramMode}}"
+              id="histogramModeSelector"
```

Comment:
Please revert all the whitespace-only changes to otherwise unrelated lines in this file.

I believe that's all the rest of the HTML changed lines except the one that does `show-download-links="[[_showDownloadLinks]]"`

Reply:
I hope I got all